packages/office-addin-dev-settings/src/dev-settings-windows.ts

```diff
@@ -71,7 +71,7 @@ export async function enableRuntimeLogging(path: string): Promise<void> {
 }
 
 export function getDeveloperSettingsRegistryKey(addinId: string): registry.RegistryKey {
-  if (!addinId) { throw new ExpectedError("addinId is required."); }
+  if (!addinId) { throw new ExpectedError("addinId is required. Manifest ID is missing."); }
```

Comment:
You just changed this before having a chance to read my comments. I don't think this is better. See my earlier comment about this.